pkg/yamlpath/example_test.go

```diff
@@ -59,7 +59,6 @@ spec:
 
 	var buf bytes.Buffer
 	e := yaml.NewEncoder(&buf)
-	defer e.Close()
```

Comment:
Why remove this from the example? It seems unrelated to this PR.

Reply:
@glyn this was causing the linter to fail due to the defer not being called if it hit the log.Fatal, it was lazy i know i can fix it up

Reply:
fixed